microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -161,16 +161,33 @@ Google may collect some data associated with the Media Router extension. To unin
 
-You can delete templates stored on the device and clear the cache data. Go to `edge://settings/privacy`. In **Clear browsing data** next to **Clear browsing data now**, select the **Choose what to clear** button. Choose your desired time range and type of data, then select the **Clear now** button.  Another way to delete cached data, go to `edge://settings/clearBrowserData` select the desired time range and type of data, then select the **Clear now** button.
 
-<!-- Above, should pick one way. Check UI for privacy updates -->
+<!-- ------------------------------ -->
+#### Signed-in with a personal account, and Sync enabled
 
-You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
-Collections aren't available when using **InPrivate** or **Guest** modes.
+When you sign in to Microsoft Edge with a personal Microsoft Account (MSA) and enable Sync, your collections, saved items, notes, and images are automatically saved on Microsoft cloud for free. You can access your collections from Microsoft Edge on any device where you're signed in with Sync enabled. You can even view and manage your collections online in any browser by visiting the [Collections portal](https://www.bing.com/saves) at Bing.com. Collections also enhance your experience with a personalized interest feed, if your personalization settings support this feature.
+
+If you want to disable Collections for a signed-in profile using a personal account with Sync enabled, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.
+
+
+<!-- ------------------------------ -->
+#### Signed-in with a work or school account, and Sync enabled
+
+If you're signed in with a work or school account and are syncing Collections, your collections, saved items, notes, and images are synced across devices. If you want to stop syncing your Collections, you can turn off Sync for Collections from `edge://settings/profiles/sync`.
+
+
+<!-- ------------------------------ -->
+#### Signed-out, or Sync disabled
+
+When you don't sign in to Microsoft Edge, or when you sign in with Sync disabled, all collections data is stored locally on the device, organized per Microsoft Edge profile.
+
+Collections aren't available when using Guest mode.
+
+When using a Microsoft Account (MSA), collections aren't available when InPrivate browsing.  InPrivate browsing is supported when using a Microsoft Azure Active Directory (AAD) account, or when not signed in.
```

Comment:
```suggestion
When using a Microsoft Account (MSA), collections aren't available when InPrivate browsing.  Collections are available when using InPrivate browsing, if you are using a Microsoft Azure Active Directory (AAD) account, or when you aren't signed in.
```
@yuanjunz - tech review this change, which shifts the emphasis from InPrivate support to Collections support.  Not sure if the existing final sentence is correct; depends on interpretation of "It is avoidable" in the notes:

Orig notes:
Collections aren't available when using Guest mode or InPrivate browsing. => expand below
Collections aren't available when using Guest mode -- for all condition
Collections aren't available when InPrivate browsing -- this is only for Microsoft account case (MSA), It is avoidable for AAD and non sign in.

Reply:
essentially done; redistributed the points.  latest view:
https://github.com/MicrosoftDocs/edge-developer/pull/2778/files